src/h5web/visualizations/heatmap/models.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  interpolateCool,
+  interpolateMagma,
+  interpolateInferno,
+  interpolatePlasma,
+  interpolateViridis,
+  interpolateBrBG,
+  interpolatePRGn,
+  interpolatePiYG,
+  interpolatePuOr,
+  interpolateRdBu,
+  interpolateRdGy,
+  interpolateRdYlBu,
+  interpolateRdYlGn,
+  interpolateSpectral,
+  interpolateBlues,
+  interpolateGreens,
+  interpolateGreys,
+  interpolateOranges,
+  interpolatePurples,
+  interpolateReds,
+  interpolateWarm,
+  interpolateBuGn,
+  interpolateBuPu,
+  interpolateGnBu,
+  interpolateOrRd,
+  interpolatePuBuGn,
+  interpolatePuBu,
+  interpolatePuRd,
+  interpolateRdPu,
+  interpolateYlGnBu,
+  interpolateYlGn,
+  interpolateYlOrBr,
+  interpolateYlOrRd,
+  interpolateRainbow,
+  interpolateSinebow,
+} from 'd3-scale-chromatic';
+
+export const COLOR_MAPPINGS: Record<string, (t: number) => string> = {
+  // Diverging
+  BrBG: interpolateBrBG,
+  PRGn: interpolatePRGn,
+  PiYG: interpolatePiYG,
+  PuOr: interpolatePuOr,
+  RdBu: interpolateRdBu,
+  RdGy: interpolateRdGy,
+  RdYlBu: interpolateRdYlBu,
+  RdYlGn: interpolateRdYlGn,
+  Spectral: interpolateSpectral,
+  // Sequential (single hue)
+  Blues: interpolateBlues,
+  Greens: interpolateGreens,
+  Greys: interpolateGreys,
+  Oranges: interpolateOranges,
+  Purples: interpolatePurples,
+  Reds: interpolateReds,
+  // Sequential (multi-hue)
+  // Turbo missing !
+  Viridis: interpolateViridis,
+  Inferno: interpolateInferno,
+  Magma: interpolateMagma,
+  Plasma: interpolatePlasma,
+  // Cividis missing !
```

Comment:
Cividis and Turbo colormaps should implemented in the version of d3-scale-chromatic that we are using ([1.5.0](https://github.com/d3/d3-scale-chromatic/releases)) but I cannot import them. Perhaps they are missing from the `@types/d3-scale-chromatic` package ?

Reply:
Indeed, `@types/d3-scale-chromatic` is for 1.3.2 : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-scale-chromatic/index.d.ts#L9